braintrust_core/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "braintrust_core",
+  "version": "0.0.1",
+  "description": "Shared core dependencies for Braintrust packages",
+  "main": "./jsdist/index.js",
+  "module": "./jsdist/index.mjs",
+  "types": "./jsdist/index.d.ts",
+  "scripts": {
```

Comment:
I would make this consistent with the `sdk/js` convention and use `dist` instead of `jsdist`. The latter is to disambiguate between python's dist, if the source files are in the same parent (in autoevals, but not in the sdk)